src/main/java/net/rcarz/jiraclient/tempo/Team.java

```diff
@@ -0,0 +1,178 @@
+package net.rcarz.jiraclient.tempo;
+
+import net.rcarz.jiraclient.JiraException;
+import net.rcarz.jiraclient.RestClient;
+import net.rcarz.jiraclient.RestException;
+import net.sf.json.JSON;
+import net.sf.json.JSONArray;
+import net.sf.json.JSONObject;
+import org.apache.commons.lang.StringUtils;
+import org.apache.http.HttpStatus;
+
+import java.util.ArrayList;
+import java.util.Iterator;
+import java.util.List;
+
+/**
+ * Created by Volha Bibik on 01/2/2019.
+ */
+public class Team {
+
+    private static final String ID = "id";
+    private static final String NAME = "name";
+    private static final String SUMMARY = "summary";
+    private static final String LEAD = "lead";
+
+    private static String getRestUri() {
+        return TempoResource.getBaseTempoTeamsUri() + "/team/";
+    }
+
+    private Integer id;
+    private String name;
+    private String summary;
+    private String lead;
+
+    public Team(Integer id, String name, String summary, String lead) {
+        this.id = id;
+        this.name = name;
+        this.summary = summary;
+        this.lead = lead;
+    }
+
+
+    public Team(JSONObject map) {
+        id = net.rcarz.jiraclient.Field.getInteger(map.get(ID));
+        name = net.rcarz.jiraclient.Field.getString(map.get(NAME));
+        summary = net.rcarz.jiraclient.Field.getString(map.get(SUMMARY));
+        lead = net.rcarz.jiraclient.Field.getString(map.get(LEAD));
+    }
+
+    public Integer getId() {
+        return id;
+    }
+
+    public void setId(Integer id) {
+        this.id = id;
+    }
+
+    public String getName() {
+        return name;
+    }
+
+    public void setName(String name) {
+        this.name = name;
+    }
+
+    public String getSummary() {
+        return summary;
+    }
+
+    public void setSummary(String summary) {
+        this.summary = summary;
+    }
+
+    public String getLead() {
+        return lead;
+    }
+
+    public void setLead(String lead) {
+        this.lead = lead;
+    }
+
+    public static List<Team> get(RestClient restclient) throws JiraException {
+        JSON response = null;
+
+        try {
+            response = restclient.get(getRestUri());
+        } catch (Exception ex) {
+            throw new JiraException("Failed to retrieve all teams", ex);
+        }
+
+        if (!(response instanceof JSONArray)) {
+            throw new JiraException("JSON payload is malformed");
+        }
+        List<Team> result = new ArrayList<Team>();
+        Iterator it = ((JSONArray) response).iterator();
+        while (it != null && it.hasNext()) {
+            result.add(new Team((JSONObject) it.next()));
+        }
+
+        return result;
+    }
+
+    public static Team get(RestClient restClient, Integer id) throws JiraException{
+        JSON response = null;
+
+        try {
+            response = restClient.get(getRestUri() + id.toString());
```

Comment:
a bit dangerous convertion to string
Better to check before usage for null - you'll be very specific on planned behavior
Cause right now you'll get NullPointerException, handle it below and everything will be okay, but someone else may decide that `catch (Exception e)` is a bad style (it actually is) and change it to another exception - after that NPE might be thrown.
Simple rule to remember - NPE is always a developer's mistake